manifests/disklist.pp

```diff
@@ -12,8 +12,8 @@
     $config = regsubst($title, '^.*@', '')
     $disk   = regsubst($title, '@[^@]*$', '')
 
-    validate_string($config)
-    validate_string($disk)
+    assert_type(String, $config)
```

Comment:
Can regsubst ever output anything else??

Reply:
Probably not. I'm not sure if this adds any benefit. But the was a validate_string before and this assert_type doesn't cause any harm.